optimade/server/routers/utils.py

```diff
@@ -78,7 +78,15 @@ def handle_response_fields(
     new_results = []
     while results:
         entry = results.pop(0)
-        new_entry = entry.dict(exclude_unset=True)
+
+        # TODO: re-enable exclude_unset when proper handling of known/unknown fields
+        # has been implemented
+        # Have to handle top level fields explicitly here for now
+        new_entry = entry.dict(exclude_unset=False)
+        for field in ("relationships", "links", "meta", "type", "id"):
+            if field in new_entry and new_entry[field] is None:
+                del new_entry[field]
```

Comment:
Unfortunately not: the problem is that we _want_ to include `None` values from all entry properties, but not from these top-level response fields. I'm sure there is a neater way of doing this, but I'd rather be explicit for now as its hopefully only a temporary "fix" before it can be dealt with properly in #504.